### DISCUSSION TITLE
Did they delete <CTRL>+K prompt bar?

### DISCUSSION THREAD

**Discussion by webgtx** [community/community#133687] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I used to use the `CTRL`+`K` prompt bar a few months ago. It seems to be gone now. Do you have any details about why?

**Comment by Sachinsingh2002** [comment]
You can find the updated list of github keyboard shortcuts here (https://docs.github.com/en/get-started/accessibility/keyboard-shortcuts)

**Reply by webgtx** [reply]
so they did delete the prompt bar? I couldn't find `ctrl` + `k` in docs

**Comment by SohanKafle** [comment]
Hi [webgtx](https://github.com/webgtx),

It seems like the "CTRL+K" prompt bar has been removed. Here are a few potential reasons and alternatives:

**1. Feature Deprecation:** GitHub might have deprecated the CTRL+K prompt bar to streamline the interface or due to low usage.
**2. Updated Shortcuts:** Sometimes, shortcuts are updated or replaced. Check GitHub’s documentation or changelog for any updates on new shortcuts or features.

**3. Alternative Methods:**
Use the quick access menu by pressing "CTRL+Shift+P" for similar functionality.
Explore the command palette in your code editor, if you're using one like Visual Studio Code.

If you need further assistance, you can also contact [GitHub Support](https://support.github.com/) for more details.

**Reply by webgtx** [reply]
Did you just copy pasted answer using LLM chatbot? @SohanKafle

**Reply by SohanKafle** [reply]
no bro i search it up on google at first to make sure about it and then gave you the details so as you can get help
